CHANGES.md

```diff
@@ -2,6 +2,11 @@
 
 This document describes the relevant changes between releases of the OCM API
 SDK.
+
+## 0.1.351
+- Update model version v0.0.296
+  - Add json annotation to `DeleteAssociatedResources` locator in account resource
```

Comment:
This doesn't seem to be a locator. Can you please check and if confirmed update the wording?

Reply:
Changed locator to parameter